daemons/attrd/attrd_corosync.c

```diff
@@ -487,8 +487,9 @@ attrd_peer_sync_response(const crm_node_t *peer, bool peer_won, xmlNode *xml)
     }
 
     // Process each attribute update in the sync response
-    for (xmlNode *child = pcmk__xe_first_child_any(xml); child != NULL;
-         child = pcmk__xe_next(child)) {
+    for (xmlNode *child = pcmk__xe_first_child(xml, NULL, NULL, NULL);
```

Comment:
One more chance to go back to the beautiful way that things once were lol... (with `pcmk__xe_first_child_any(xml)` and adding a new `pcmk__xe_first_child_named(xml, name)`)